.travis.yml

```diff
@@ -2,23 +2,44 @@ language: go
 
 os:
   - linux
-  - osx
 
 go:
   - 1.8.3
 
 go_import_path: github.com/kubeapps/kubeapps
 
+env:
+  global:
+    VERSION=${TRAVIS_TAG:-build-$TRAVIS_BUILD_ID}
+
+matrix:
+  include:
+    - env: TARGET=x86_64-w64-mingw32
+    - env: TARGET=""
+addons:
+  apt:
+    packages:
+      # cross compiling for windows
+      - mingw-w64
+
 before_install:
   - set -e
 
 script:
   - make test
-  - make VERSION=${TRAVIS_TAG:-build-$TRAVIS_BUILD_ID} binary
+  - |
+    if [ -n "$TARGET" ]; then
```

Comment:
we should make this explicitly `if [ $TARGET = "x86_64-w64-mingw32" ]` as presumably we'll need another condition for darwin later

Reply:
sure. Will change it when the osx build is added